src/webui/www/private/views/preferences.html

```diff
@@ -356,10 +356,10 @@
 
     <fieldset class="settings">
         <legend>
-            <input type="checkbox" id="ipfilter_enabled_checkbox" onclick="qBittorrent.Preferences.updateFilterSettings();" />
-            <label for="ipfilter_enabled_checkbox">QBT_TR(IP Filtering)QBT_TR[CONTEXT=OptionsDialog]</label>
+            <label>QBT_TR(IP Filtering)QBT_TR[CONTEXT=OptionsDialog]</label>
         </legend>
         <div class="formRow">
+            <input type="checkbox" id="ipfilter_enabled_checkbox" onclick="qBittorrent.Preferences.updateFilterSettings();" />
```

Comment:
Only moving it doesn't seem right...

Reply:
Only modify the id of the checkbox.

Do you mean there are some other codes need to be modified? It seems that this checkbox only determines  whether the ipfilter file will be enabled. Correct me if I'm wrong. Thank you.